agent/stats/utils_unix.go

```diff
@@ -49,7 +49,7 @@ func getMemUsage(mem types.MemoryStats) uint64 {
 			return mem.Usage - v
 		}
 	}
-	if v, ok := mem.Stats["cache"]; ok && v < mem.Usage {
```

Comment:
There is some failed unit test e.g. `TestDockerStatsToContainerStatsMemUsage`  failed due to this change. Please update accordingly.

Reply:
Thanks. I fixed it.